src/components/Navbar/Navbar.js

```diff
@@ -1,12 +1,35 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import "./navbar.css";
 
 const Navbar = () => {
 	return (
-		<div>
-			<Link to="/tasks">Task</Link>
-			<Link to="/timer">Timer</Link>
-			<Link to="/settings">Settings</Link>
+		<div className="pomo-navbar-container">
+			<div className="brand-details">
+				<Link to="/">
+					<span className="brand-name">pomoDone!</span>
+				</Link>
+			</div>
+			<div className="navbar-links">
+				<Link to="/tasks">
```

Comment:
Maybe you can change this to NavLink so that you can accommodate active classes as well